dp-identity-api.nomad

```diff
@@ -11,70 +11,6 @@ job "dp-identity-api" {
     auto_revert      = true
   }
 
-  group "web" {
-    count = "{{WEB_TASK_COUNT}}"
-
-    constraint {
-      attribute = "${node.class}"
-      value     = "web"
-    }
-
-    restart {
-      attempts = 3
-      delay    = "15s"
-      interval = "1m"
-      mode     = "delay"
-    }
-
-    task "dp-identity-api-web" {
```

Comment:
Similar to this the task name needs to be changed in publishing group. ` task "dp-identity-api-publishing" {`  
to` task "dp-identity-api" {`

Reply:
👍